sdk/nodejs/appservice/kind.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export module Kinds {
```

Comment:
The linter says `The internal 'module' syntax is deprecated, use the 'namespace' keyword instead.` Should we use `namespace`?

Reply:
I am following the normal convention we use here - we use module everywhere else, we would need to upgrade those all at the same time - maybe a question for @CyrusNajmabadi for the future